dev/io.openliberty.microprofile.openapi.2.0.internal/src/com/ibm/ws/microprofile/openapi/servlet/filter/OpenAPIEndpointFilter.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+
+package com.ibm.ws.microprofile.openapi.servlet.filter;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import io.openliberty.microprofile.openapi.internal.common.services.OpenAPIEndpointProvider;
+import org.osgi.framework.BundleContext;
+import org.osgi.util.tracker.ServiceTracker;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+/**
+ * Filter for altering the openapi path defined in the OpenAPI UI index.html
+ *
+ *
+ */
+public class OpenAPIEndpointFilter implements Filter {
+
+    private static final TraceComponent tc = Tr.register(OpenAPIEndpointFilter.class);
```

Comment:
We have two copies of this class. Should it go in the OpenAPI UI bundle instead so we'd only need one copy?

Reply:
Ah, possibly we've done it this way around so that we don't need to worry about transforming the UI bundle to work with Jakarta. It does look like we already have another (identical) Filter in both projects.